14/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -18,6 +18,8 @@ else
     export POSTGRESQL_EFFECTIVE_CACHE_SIZE=${POSTGRESQL_EFFECTIVE_CACHE_SIZE:-$effective_cache}
 fi
 
+export POSTGRESQL_LOG_DESTINATION=${POSTGRESQL_LOG_DESTINATION:-}
```

Comment:
Could you please add distgen-generated content in a separate commit?


Also, why is common.sh adjusted only in version 14? Do you think that some of previous, already merged, commits were not dg-generated properly?

Reply:
It seems that the content for 14 was not generated correctly in one of the previous commits. So @Lupphes please rebase your PR. It should fix it.